dynamic_rest/fields/content_type.py

```diff
@@ -0,0 +1,91 @@
+import traceback
+
+from dynamic_rest.fields.common import WithRelationalFieldMixin
+from dynamic_rest.fields.fields import DynamicField
+from dynamic_rest.routers import DynamicRouter
+from dynamic_rest.tagged import TaggedDict
+
+
+class DynamicGenericRelationField(
+    WithRelationalFieldMixin,
+    DynamicField
+):
+
+    def __init__(self, embed=False, *args, **kwargs):
+        super(DynamicGenericRelationField, self).__init__(*args, **kwargs)
+        self.embed = embed
+        self.read_only = True
+
+    def bind(self, field_name, parent):
+        super(DynamicGenericRelationField, self).bind(field_name, parent)
+
+        source = self.source or field_name
+
+        # Inject `requires` so required fields get prefetched properly.
+        # TODO: It seems like we should be able to require the type and
+        #       id fields, but that seems to conflict with some internal
+        #       Django magic. Disabling `.only()` by requiring '*' seem
+        #       to work more reliably...
+        self.requires = [
```

Comment:
What if `requires` is specified manually as well? Should we concatenate, or is it safe to over-write here?